code/modules/cm_marines/altitude_control_console.dm

```diff
@@ -100,35 +111,38 @@ GLOBAL_VAR_INIT(ship_alt, SHIP_ALT_MED)
 
 	if(.)
 		return
-	var/mob/user = usr
-	switch(action)
-		if("low_alt")
-			change_altitude(user, SHIP_ALT_LOW)
-			. = TRUE
-		if("med_alt")
-			change_altitude(user, SHIP_ALT_MED)
-			. = TRUE
-		if("high_alt")
-			change_altitude(user, SHIP_ALT_HIGH)
-			. = TRUE
-	message_admins("[key_name(user)] has changed the ship's altitude to [action].")
-
-	add_fingerprint(usr)
+	var/mob/user = ui.user
+	if(TIMER_COOLDOWN_CHECK(src, COOLDOWN_ALTITUDE_CHANGE))
+		message_admins("[key_name(user)] tried to change the ship's altitude, but it is still on cooldown.")
+		to_chat(user, SPAN_WARNING("Engines pending recalibration to burn again, stand by."))
+	else
+		switch(action)
+			if("low_alt")
+				change_altitude(user, SHIP_ALT_LOW)
+				. = TRUE
+			if("med_alt")
+				change_altitude(user, SHIP_ALT_MED)
+				. = TRUE
+			if("high_alt")
+				change_altitude(user, SHIP_ALT_HIGH)
+				. = TRUE
+		message_admins("[key_name(user)] has changed the ship's altitude to [action].")
+
+	add_fingerprint(ui.user)
 
 /obj/structure/machinery/computer/altitude_control_console/proc/change_altitude(mob/user, new_altitude)
-	if(TIMER_COOLDOWN_CHECK(src, COOLDOWN_ALTITUDE_CHANGE))
-		to_chat(user, SPAN_WARNING("The engines are not ready to burn yet."))
-		return
 	if(GLOB.ship_alt == new_altitude)
 		return
-	GLOB.ship_alt = new_altitude
-	TIMER_COOLDOWN_START(src, COOLDOWN_ALTITUDE_CHANGE, 90 SECONDS)
-	for(var/mob/living/carbon/current_mob in GLOB.living_mob_list)
-		if(!is_mainship_level(current_mob.z))
-			continue
-		current_mob.apply_effect(3, WEAKEN)
-		shake_camera(current_mob, 10, 2)
-	ai_silent_announcement("Attention: Performing high-G manoeuvre", ";", TRUE)
+	else
```

Comment:
This else can be removed and everything under it tabbed over because of the early return above.